### ISSUE TITLE
Migrate command does not follow the migration_auto_latest config

### ISSUE THREAD

**Issue by albertleao** [kenjis/codeigniter-cli#7]
When your config/migrations.php has the following:

`$config['migration_auto_latest'] = TRUE;`

The command 'php cli migrate' still does not perform the migrations unless the version number is set.

**Comment by anhnnp** [comment]
Yes, my issue same your question.  I set `$config['migration_auto_latest'] = TRUE;` and run `php cli migrate` not update version lasted.
And how to update table using cli migration? Example: I created User table migration and now I want alter column table or add new column into User table.
